packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -965,6 +984,21 @@ export class FieldEditor {
     );
   }
 
+  private renderDependentDropdown(boolDisableDropdowns) {
+    const objFormValue = this.fieldBuilderOptions;
+
+    return (
+      <fw-fb-field-dropdown
+        ref={(el) => (this.dictInteractiveElements['choices'] = el)}
```

Comment:
Can we change the ref name to "dependent_dropdown" or something instead of "choices" here pls